jupyter_elfinder/elfinder.py

```diff
@@ -508,138 +549,138 @@ def __file(self) -> None:
         self.http_header["Content-Transfer-Encoding"] = "binary"
         self.http_header["Content-Length"] = str(os.lstat(cur_file).st_size)
         self.http_header["Connection"] = "close"
-        self._response["file"] = cur_file
+        self._response[R_FILE] = cur_file  # FIXME: This is not part of api 2.1
```

Comment:
I think this is not part of api 2.1. See https://github.com/Studio-42/elFinder/wiki/Client-Server-API-2.1#file

Reply:
This is a temporary variable which we used in https://github.com/imjoy-team/jupyter-elfinder/blob/2248057290b1b9784bd537cf0a3ac0d3fa3d7aed/jupyter_elfinder/views.py#L120 to send the file.